docs/Features/Keyboard.md

```diff
@@ -4,11 +4,15 @@ Keyboard handling is an area that is constantly seeing improvements and bug fixe
 That's because each platform does things slightly differently and xpra has to somehow convert this data into meaningful keyboard events on the remote end.
 
 ## Common Issues
-* when connecting over high latency links, use the `--no-keyboard-sync` option to prevent keys from repeating. This toggle is also accessible from the system tray menu. (this switch may cause other problems though)
-* keyboard shortcuts: [#1657](https://github.com/Xpra-org/xpra/issues/1657)
+* When connecting over high latency links, use the `--no-keyboard-sync` option to prevent keys from repeating. This toggle is also accessible from the system tray menu. (this switch may cause other problems though)
+* Keyboard shortcuts: You can find them ...
+  * from the Tray Icon > Right Click > Keyboard > View Shortcuts
+  * by pressing `prefix+F6` (by default, `prefix` is also by default set to `meta+shift`)
```

Comment:
This is incorrect, the prefix depends on the OS and DE, this is explained elsewhere.

Reply:
The system tray documentation says: _On such platforms, users may have to use the `#+F1` to bring up xpra's menu (typically that's `Control+F1` or `Alt+F1` on some platforms) or force enable the headerbar (`headerbar=force`)._

Reply:
I took ... quite a stab on improving it 😅 

I hope you approve 🙏 (of the effort. The MR we can discuss of course 🤪)